Examples/Io/Csv/src/CsvPlanarClusterReader.cpp

```diff
@@ -79,16 +79,7 @@ struct CompareHitId {
 /// Convert separate volume/layer/module id into a single geometry identifier.
 inline Acts::GeometryIdentifier extractGeometryId(
     const ActsExamples::HitData& data) {
-  // if available, use the encoded geometry directly
```

Comment:
As the PlanarClusterReader will vanish, this fulfills the right functionality now, but I didn't really do it cleanly as this code will go dead soon ...